tests/index.js

```diff
@@ -815,6 +815,14 @@ describe('Thorn', () => {
             });
 
             it('should use internal usernames for server interactions', function*() {
+                let uuidv4Format = [
+                    '[a-fA-F0-9]{8}',
+                    '[a-fA-F0-9]{4}',
+                    '4{1}[a-fA-F0-9]{3}',
+                    '[89abAB]{1}[a-fA-F0-9]{3}',
```

Comment:
For clarification: the 89AB is the `variant` part of the RFC 4122 UUID. Per [the spec](http://www.ietf.org/rfc/rfc4122.txt):

```
Msb0  Msb1  Msb2  Description
    0     x     x    Reserved, NCS backward compatibility.
    1     0     x    The variant specified in this document.
    1     1     0    Reserved, Microsoft Corporation backward compatibility
    1     1     1    Reserved for future definition.
```

8, 9, A, and B are the only hexadecimal digits that start with `10` in binary; anything higher than that starts with `11` and anything lower than that starts with `01`.